src/main/java/me/itzg/tsdbcassandra/web/ParamUtils.java

```diff
@@ -0,0 +1,31 @@
+package me.itzg.tsdbcassandra.web;
+
+import java.util.List;
+import org.springframework.util.MultiValueMap;
+
+public class ParamUtils {
+
+  /**
+   * The OpenTSDB APIs use a convention where a query parameter that is present, but doesn't have
+   * any value indicates that it is "enabled". Given a {@link MultiValueMap} obtained using
+   * {@link org.springframework.web.bind.annotation.RequestParam}, this method will a parameter
```

Comment:
>this method will 'interpret'? a parameter

Reply:
Wow, I can't tell what phrasing I meant there, but "interpret" does seem to be the missing word.